warehouse/templates/manage/organization/activate_subscription.html

```diff
@@ -11,28 +11,47 @@
  # See the License for the specific language governing permissions and
  # limitations under the License.
 -#}
-{% extends "confirm-action.html" %}
+{% extends "manage_organization_base.html" %}
 
 {% block title %}
   {% trans %}Activate Subscription{% endtrans %}
 {% endblock %}
 
-{% block prompt %}
-  {% trans %}Activate Subscription{% endtrans %}
-{% endblock %}
 
 {% block main %}
-<div class="form-group">
-  <span class="form-group__text">
-    {% trans %}Company accounts require an active subscription. Please enter up-to-date billing information to enable the account.{% endtrans %}
-  </span>
-</div>
-<div>
-  <a href="{{ request.route_path('manage.organizations') }}" class="button">
-    {% trans %}Cancel{% endtrans %}
-  </a>
-  <a href="{{ request.route_path('manage.organization.subscription', organization_name=organization.normalized_name) }}" class="button button--primary">
-    {% trans %}Activate Subscription{% endtrans %}
-  </a>
-</div>
+  <h2>{% trans %}Activate Subscription{% endtrans %}</h2>
+  <form method="POST" data-controller="organization-terms-of-service-accepted">
+    <div class="form-group__wide">
+      <p class="form-group__text">
+        {% trans %}Company accounts require an active subscription. Please enter up-to-date billing information to enable the account.{% endtrans %}
+      </p>
+    </div>
+    <input name="csrf_token" type="hidden" value="{{ request.session.get_csrf_token() }}">
+    {{ form_errors(form) }}
+    <div class="form-group__wide">
+      <label for="terms_of_service_agreement" class="form-group__label">
+        {% trans %}Terms of Service{% endtrans %}
+        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
+      </label>
+      <input type="checkbox" name="terms_of_service_agreement" data-action="input->organization-terms-of-service-accepted#checkTermsOfServiceAccepted" data-organization-terms-of-service-accepted-target="termsOfServiceAccepted" />
+      {% trans tos_url="https://policies.python.org/pypi.org/Terms-of-Service/", organization_name=organization.display_name %}
```

Comment:
This URL is 404 right now but I assume you're aware of that!

Reply:
Confirmed. It will be live before we go approving any orgs in prod 🫡